maven-enforcer-rules/README.md

```diff
@@ -0,0 +1,51 @@
+# Helidon Build Common Maven Enforcer Rules
+
+This module provides common Helidon enforcer rules intended to integrate with the [Maven Enforcer Plugin](https://maven.apache.org/enforcer/maven-enforcer-plugin/).
+
+## Rules
+* [HelidonDependenciesRule](src/main/java/io/helidon/build/maven/enforcer/rules/HelidonDependenciesRule.java) - Verifies compile and runtime maven dependencies (including transitive dependencies) for compatibility with Helidon.
+
+### General usage
+
+Here is an example pom.xml:
+
+```xml
+<build>
+    <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-enforcer-plugin</artifactId>
+            <dependencies>
+                <dependency>
+                    <groupId>io.helidon.build-tools</groupId>
+                    <artifactId>helidon-build-maven-enforcer-rules</artifactId>
+                    <version>${helidon.build-tools.version}</version>
+                </dependency>
+            </dependencies>
+            <executions>
+                <execution>
+                    <id>enforce-helidon-dependencies</id>
+                    <goals>
+                        <goal>enforce</goal>
+                    </goals>
+                    <configuration>
+                        <rules>
+                            <helidonDependenciesRule>
+                                <namespace>JARKTA | JAVAX</namespace>
```

Comment:
`JARKTA` -> `JAKARTA`, also this should probably be documented as lowercase, even if the string is case insensitive.

Reply:
fixed